Dockerfile

```
@@ -1,48 +1,22 @@
-# capnp
-FROM alpine as capnp
-
-RUN mkdir -p /src
-RUN apk update && apk add autoconf automake libtool linux-headers git g++ make
-RUN cd /src && git clone https://github.com/capnproto/capnproto.git
-WORKDIR /src/capnproto/c++
-RUN ./setup-autotools.sh
-RUN autoreconf -i
-RUN ./configure
-RUN make -j6 check
-RUN make install
-RUN which capnp
-
 # base
-FROM ocaml/opam2:alpine as base
-RUN sudo apk add --update m4 gmp gmp-dev perl
+FROM ocaml/opam2:debian-stable as base
```

```suggestion
FROM ocaml/opam as base
```

This should use `ocaml/opam` now, as ocaml/opam2 is being deprecated.